docs/releases/v0.25.0.md

```diff
@@ -0,0 +1,20 @@
+# Changelog
+
+## [v0.25.0](https://github.com/aws-observability/aws-otel-collector/tree/v0.25.0) (2023-01-05)
+
+[Full Changelog](https://github.com/aws-observability/aws-otel-collector/compare/v0.23.1...v0.25.0)
```

Comment:
```suggestion
[Full Changelog](https://github.com/aws-observability/aws-otel-collector/compare/v0.24.1...v0.25.0)
```

Reply:
Thanks very much. I missed that. Updated now.